pkg/vsphere/tasks/waiter.go

```diff
@@ -63,22 +62,45 @@ func Wait(ctx context.Context, f func(context.Context) (Waiter, error)) error {
 //       return vm.Reconfigure(ctx, config)
 //    })
 func WaitForResult(ctx context.Context, f func(context.Context) (ResultWaiter, error)) (*types.TaskInfo, error) {
-	task, err := f(ctx)
+	r := rand.New(rand.NewSource(time.Now().UnixNano())) //creates a more unique random
```

Comment:
'1' is just the seed, as time.Now().  With the use of 'r.Int63n(100)' here, you're picking a random number in the range of 0..100, so the seed won't make much difference in this case.  Why add code we don't need?